crashreports/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+set(TARGET crashreports)
```

Comment:
I'd like this new CMakeList.txt file (and all new ones) to have comments at top explaining the purpose.  Perhaps in
#[[
long form
]]#

Reply:
So this defines a special library statically linked into Audacity, which uses breakpad::client, which is -- shared or static?

I was thinking maybe these files should just be part of src/, but then I reconsidered the special need to have an easier build-time off switch for the whole thing, so this organization may be better.

So let's explain all that in comments for the next reader.

Reply:
Explain too how parts execute at startup, others in the immediate post-crash context.